### PR TITLE
ISSD-2293 [CP 2.0] Release Notes page - Navigation and Hero Image

### PR DESCRIPTION
**Pull request by samuelleolopes**
[ISSD-2293](https://liferay.atlassian.net/browse/ISSD-2293)

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	<#if quarterlyReleaseVocabularyId?has_content>
+		<#assign releaseCategories = (restClient.get("/headless-admin-taxonomy/v1.0/taxonomy-vocabularies/${quarterlyReleaseVocabularyId}/taxonomy-categories?pageSize=4&sort=dateCreated:desc").items)! />
+	</#if>
+</#if>
+
+<style>
+	#dropdownReleaseNotes {
+		font-size: var(--h2-font-size, 1.375rem) !important;
+	font-weight: var(--h2-font-weight) !important;
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl:12]: indent needed here

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	}
+</style>
+
+<div class="dropdown">
+	<button
+		aria-expanded="false"
+		aria-haspopup="true"
+		class="btn p-0 text-neutral-0 dropdown-toggle"
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl:20]: sort css classes in alphabetical order

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+			${AssetCategory_name.getData()}
+		</#if>
+
+		<@clay["icon"] symbol="caret-bottom" />
+	</button>
+
+	<ul
+		aria-labelledby="dropdownReleaseNotes"
+		class="dropdown-menu"
+		x-placement="bottom-start"
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl:35]: sort

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/dxp-release-notes/page-definition.json [hunk 4]
```diff
@@ -0,0 +1,561 @@
...
+								"contentDisplay": "FlexColumn",
+								"justify": "Center",
+								"marginRight": 4
+							}
+						},
+						"id": "d5f16230-a734-cc7d-779e-9907a365093a",
+						"pageElements": [
+							{
+								"definition": {
+									"customCSS": ".[$FRAGMENT_CLASS$] .btn {\n    color: var(--color-brand-primary) !important;\n    background-color: var(--color-neutral-0) !important;\n    border-color: var(--color-brand-primary) !important;\n}\n\n.[$FRAGMENT_CLASS$] p {\n    margin-bottom: 0;\n}",
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layout-page-templates/display-page-templates/dxp-release-notes/page-definition.json:129]: sort

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+		style="top: 0px; transform: translate3d(0px, 40px, 0px);"
+	>
+		<#if releaseCategories?has_content>
+			<#list releaseCategories as releaseCategory>
+				<#assign friendlyURL = (releaseCategory.taxonomyCategoryProperties?filter(taxonomyCategoryProperty -> stringUtil.equals(taxonomyCategoryProperty.key, "friendlyURL"))?first.value)! />
+
+			<li>
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl:42]: indent

**Comment by samuelleolopes** [comment]
I'm going to open another PR. Updated and synchronized. @kylembischof @jairmedeiros

**Comment by samuelleolopes** [comment]
reopened https://github.com/liferay-customer-delivery/liferay-portal/pull/8

**Pull request closed without merging**